tools/template.xml.in

```diff
@@ -1,113 +1,120 @@
+      <master startport="4" />
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x1d" function="0x2" />
     </controller>
-    <controller type='sata' index='0'>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x1f' function='0x2'/>
+    <controller type="sata" index="0">
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x1f" function="0x2" />
     </controller>
-    <interface type='network'>
-      <mac address='52:54:00:92:d4:7b'/>
-      <source network='default'/>
-      <model type='e1000-82545em'/>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x02' function='0x0'/>
+    <interface type="user">
+      <mac address="52:54:00:92:d4:7b" />
+      <model type="e1000-82545em" />
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x02" function="0x0" />
     </interface>
-    <serial type='pty'>
-      <target type='isa-serial' port='0'>
-        <model name='isa-serial'/>
+    <serial type="pty">
+      <target type="isa-serial" port="0">
+        <model name="isa-serial" />
       </target>
     </serial>
-    <console type='pty'>
-      <target type='serial' port='0'/>
+    <console type="pty">
+      <target type="serial" port="0" />
     </console>
-    <input type='mouse' bus='usb'>
-      <address type='usb' bus='0' port='1'/>
+    <input type="tablet" bus="usb">
+      <address type="usb" bus="0" port="1" />
     </input>
-    <input type='keyboard' bus='usb'>
-      <address type='usb' bus='0' port='2'/>
+    <input type="keyboard" bus="usb">
+      <address type="usb" bus="0" port="2" />
     </input>
-    <graphics type='spice' autoport='yes'>
-      <listen type='address'/>
-      <image compression='off'/>
+    <graphics type="spice">
+      <listen type="none" />
+      <image compression="off" />
+      <gl enable="no" />
     </graphics>
-    <sound model='ich9'>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x1b' function='0x0'/>
+    <sound model="ich9">
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x1b" function="0x0" />
     </sound>
     <video>
-      <model type='qxl' ram='65536' vram='65536' vgamem='16384' heads='1' primary='yes'/>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x01' function='0x0'/>
+      <model type="vga" vram="131072" heads="1" primary="yes" />
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x01" function="0x0" />
     </video>
-    <redirdev bus='usb' type='spicevmc'>
-      <address type='usb' bus='0' port='3'/>
+    <redirdev bus="usb" type="spicevmc">
+      <address type="usb" bus="0" port="3" />
     </redirdev>
-    <redirdev bus='usb' type='spicevmc'>
-      <address type='usb' bus='0' port='4'/>
+    <redirdev bus="usb" type="spicevmc">
+      <address type="usb" bus="0" port="4" />
     </redirdev>
-    <memballoon model='none'/>
+    <memballoon model="none" />
   </devices>
+  <seclabel type="dynamic" model="selinux" relabel="yes" />
```

Comment:
This leads to failure if `selinux` is not enabled on the host system. In Debian, for example, SELinux is not installed by default.

Reply:
It could be script-fixable

Reply:
@kowalski7cc I would recommend removing this line or ensuring it can be dynamically added by the script (e.g. "Do you use SELinux? y / [n]")